test/test_talker_listener.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2016 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from functools import partial
+from multiprocessing import Process
+from time import sleep
+
+
+def talker():
+    import rclpy
+    from rclpy.qos import qos_profile_default
```

Comment:
In my tests with multiprocessing, the native extensions were not properly loaded in the subprocess, so I ended up doing the imports locally. I'll run them again and try to fix it, I opened an ticket to investigate the issue: https://github.com/ros2/rclpy/issues/4